quay-enterprise/quay-ssl.md

```diff
@@ -0,0 +1,136 @@
+Organizational Unit Name (eg, section) []:Demo Quay
+Common Name (eg, your name or your server's hostname) []:lan-lab-07.lab.libcore.so
+Email Address []:support@quay.io
+```
+
+Sign the certificate with the CA:
+
+```
+$ openssl x509 -req -in ssl.csr -CA rootCA.pem -CAkey rootCA.key -CAcreateserial -out ssl.cert -days 500 -sha256
+Signature ok
+subject=/C=US/ST=California/L=SF/O=Demo Quay/OU=Demo Quay/CN=lan-lab-07.lab.libcore.so/emailAddress=support@quay.io
+Getting CA Private Key
+```
+The next step can be accomplished via the QE superuser panel or from the terminal:
+
+## To configure with the superuser GUI in QE:
+
+Set the `Server Hostname` to the appropriate value and check the `Enable SSL`:
+
+<img src="img/server-hostname.png" class="img-center" alt="Set server hostname"/>
+
+Upload the `ssl.key` and `ssl.cert` files.
+
+<img src="img/upload-cert.png" class="img-center" alt="Upload Certificate"/>
+
+Save the configuration. QE will automatically validate the SSL certificate.
+
+<img src="img/save-configuration.png" class="img-center" alt="Save and Check config"/>
+
+Restart the container:
+
+<img src="img/restart-container.png" class="img-center" alt="Restart Container"/>
+
+
+## Setting up in the terminal
+
+By not using the web interface the configuration checking mechanism built into QE is unavailable. It is suggested to use the web interface if possible.
+
+Copy the `ssl.key` and `ssl.cert` into the specified `config` directory.
+
+**Note: The certificate/key files must be named ssl.key and ssl.cert**
+
+```
+$ ls
+ssl.cert  ssl.key
+
+$ scp ssl* core@10.7.8.117:/home/core/config/
+
+core@lan-lab-7 ~ $ ls config/
+config.yaml  ssl.cert  ssl.key
+```
+
+Modify the `PREFERRED_URL_SCHEME:` parameter in config.yaml from `http` to `https`
+
+```
+PREFERRED_URL_SCHEME: https
+```
+
+Restart the QE container:
+
+```
+$ docker ps
+CONTAINER ID        IMAGE                     COMMAND                  CREATED             STATUS              PORTS                                                NAMES
+eaf45a4aa12d        quay.io/quay/redis        "/usr/bin/redis-serve"   22 hours ago        Up 22 hours         0.0.0.0:6379->6379/tcp                               dreamy_ramanujan
+cbe7b0fa39d8        quay.io/coreos/registry   "/sbin/my_init"          22 hours ago        Up About an hour    0.0.0.0:80->80/tcp, 0.0.0.0:443->443/tcp, 8443/tcp   fervent_ptolemy
+705fe7311940        mysql:5.7                 "/entrypoint.sh mysql"   23 hours ago        Up 22 hours         0.0.0.0:3306->3306/tcp                               mysql
+
+$ docker restart cbe7b0fa39d8
+```
+
+# Test out the secure connection
+
+Confirm the configuration by visiting the URL from a browser `https://lan-lab-07.lab.libcore.so/`
+
+<img src="img/https-browser.png" class="img-center" alt="Browser"/>
+
+"Your Connection is not secure" means the CA is untrusted but confirms that SSL is functioning properly. Google how to trust a CA based
```

Comment:
I need help with this line. 

"Google how to trust a CA for the browser/operating system you are running."
"Google how to trust a CA based on the browser/operating system."
Remove the line?